hugo/content/docs/getting_started.md

```diff
@@ -0,0 +1,263 @@
+
+## Connect to a model source
+Sprotty supports two kinds of model sources:
+
+* LocalModelSource allows to create models directly in TypeScript or JavaScript.
+* WebSocketDiagramServer delegates to a remote source that is connected via web socket.
+
+In this example, we consider the local variant. To enable the model source, we add the following line to our module definition (see previous section):
+```ts
+bind(TYPES.ModelSource).to(LocalModelSource).inSingletonScope()
+```
+Afterwards you can use the LocalModelSource to initialize and update the model. For example, the following `graph` consists of three task nodes with a connection between the first two. Create a new file `model-source.ts` at the root of the project:
+```typescript
+import { SGraph, SEdge, SNode } from "sprotty-protocol";
+import { TaskNode } from "./models";
+
+export const graph: SGraph = {
+    type: 'graph',
+    id: 'graph',
+    children: [
+        <SNode & TaskNode>{
+            type: 'task',
+            id: 'task01',
+            name: 'First Task',
+            isFinished: true,
+            isRunning: false,
+            position: { x: 0, y: 0 },
+            size: { width: 100, height: 100 }
+        },
+        <SNode & TaskNode>{
+            type: 'task',
+            id: 'task02',
+            name: 'Second Task',
+            isFinished: false,
+            isRunning: true,
+            position: { x: 0, y: 200 },
+            size: { width: 100, height: 100 }
+        },
+        <SNode & TaskNode>{
+            type: 'task',
+            id: 'task03',
+            name: 'Third Task',
+            isFinished: false,
+            isRunning: false,
+            position: { x: 150, y: 0 },
+            size: { width: 100, height: 100 }
+        },
+        <SEdge>{
+            type: 'edge',
+            id: 'edge01',
+            sourceId: 'task01',
+            targetId: 'task02',
+            routerKind: 'manhattan',
+        }
+    ]
+};
+```
+
+Finally, we need to create the entry point of our application in a `index.ts` file:
+```typescript
+import "reflect-metadata";
+import { LocalModelSource, TYPES } from 'sprotty';
+import { createContainer } from './di.config';
+import { graph } from './model-source';
+
+export default function run() {
+    const container = createContainer("sprotty-container");
+    const modelSource = container.get<LocalModelSource>(TYPES.ModelSource);
+    modelSource.setModel(graph);
+}
+
+document.addEventListener("DOMContentLoaded", () => run());
+```
+
+That's it! Run `npm run build` and open your HTML file, you should see the following diagram! There is some level of interactivity by default, try it out! Select nodes by clicking on them and move them around by dragging, adjust the zoom-level with the mouse wheel, navigate the diagram by panning via left-click and drag outside of a node.
+![Getting started diagram](/getting-started-diagram.png)
```

Comment:
Recommended next steps?

Reply:
I think I will add this later on when we have decided on the order of the documentation.